scripts/scripts.js

```diff
@@ -278,13 +278,13 @@ export async function getAllBlogs(category) {
     if (a.lastModified < b.lastModified) return 1;
     return 0;
   });
-
   // move featured article to the top of the sorted list
-  const featuredArticleIndex = blogArticles.findIndex((el) => (el.featuredArticle === 'true'));
-  const featuredArticle = blogArticles[featuredArticleIndex];
-  blogArticles.splice(featuredArticleIndex, 1);
-  blogArticles.unshift(featuredArticle);
-
+  const featuredArticleIndex = blogArticles.findIndex((el) => (el['featured-article'] === 'true'));
+  if (featuredArticleIndex > -1) {
```

Comment:
`>= 0`?

Reply:
I was trying to ask him about that on our call, but he misunderstood me 😃